Documentation/commands.md

```diff
@@ -108,8 +108,19 @@ UUID					APP	IMAGE NAME		IMAGE ID		STATE	NETWORKS
 The pod's machine name will be the pod's UUID prefixed with `rkt-`.
 Given this machine name, logs can be retrieved by `journalctl`:
 
+##### Running container
```

Comment:
please just remove this line, it's stated on line 90 that it's for a running pod